### ISSUE TITLE
Unable to update to 2.2.3

### ISSUE THREAD

**Issue by davewatson91** [AndrewPaglusch/FlashPaper#145]
Hi mate, 

Firstly, really love your work - it's much appreciated. 

When trying to update to your latest release, I'm hitting a snag trying to pull it - getting an error message: 
`no matching manifest for linux/amd64 in the manifest list entries`

I can see that you have arm64 mentioned in the repo, but that's it - is there something I'm missing here? 

Thanks! 
Dave

**Comment by AndrewPaglusch** [comment]
Hey @davewatson91, thanks for the kind words! Let me take a look into this and follow up with you. This is probably related to the changes I just introduced in #143

**Comment by AndrewPaglusch** [comment]
I was able to reproduce this:
```
$ docker-compose pull <containername>
[...]
no matching manifest for linux/amd64 in the manifest list entries
```

**Comment by AndrewPaglusch** [comment]
This should be fixed now from #146 and #147. It looks like I was doing multi-architecture builds incorrectly. Can you try pulling the latest v2 tag, or v2.2.4? Thanks!

**Comment by davewatson91** [comment]
> This should be fixed now from #146 and #147. It looks like I was doing multi-architecture builds incorrectly. Can you try pulling the latest v2 tag, or v2.2.4? Thanks!

Works perfectly - thanks for the super quick response!

**Comment by AndrewPaglusch** [comment]
@davewatson91 Thanks for following up and for bringing this to my attention! I'm happy to hear that it's working again for you.
